examples/hotkey_mapper.html

```diff
@@ -1,45 +1,61 @@
-<!doctype html>
```

Comment:
there's no functional change — this is just the result form running prettier on this file.